elk/utils_evaluation/parser.py

```diff
@@ -67,13 +66,20 @@ def get_args(default_config_path=Path(__file__).parent / "default_config.json"):
     )
     args = parser.parse_args()
 
+    # Dereference shortcut
+    args.model = model_shortcuts.get(args.model, args.model)
```

Comment:
Why the double args.model?

Reply:
`.get()` takes an optional second parameter which is the default value returned when the key doesn't exist in the dictionary. So if the model string isn't in the shortcuts dictionary, it's interpreted as a raw HF model string